plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsListenersTest.kt

```diff
@@ -21,7 +22,7 @@ import org.robolectric.annotation.LooperMode
 import org.robolectric.shadows.ShadowLog
 
 @RunWith(RobolectricTestRunner::class)
-@Config(shadows = [ShadowLogger::class])
+@Config(sdk = [Build.VERSION_CODES.O], shadows = [ShadowLogger::class])
```

Comment:
Could you please clarify a bit why O version here and P below? Does it remove some warnings?

Reply:
should have been O across the board, fixed.